sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -749,6 +749,12 @@ object GpuOverrides extends Logging {
    * @param meta agg expression meta
    */
   def checkAndTagAnsiAgg(checkType: Option[DataType], meta: AggExprMeta[_]): Unit = {
+    if (meta.expr.isInstanceOf[Count]) {
```

Comment:
This is not good. 
I would prefer to overriding `needsAnsiCheck` to false in the Rapids meta class of `Count`.

Reply:
And also confused why these tests fail on Spark 400, since ansi mode is already expplicitly enabled in these tests.